### PR TITLE
[WIP] Code cleanup

### PR DESCRIPTION
**Pull request by englehardt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +149,7 @@ available [below](#output-format).
         browser is running on the machine.
 * Cookie Access
     * Set `browser_params['cookie_instrument'] = True`
-    * Data is saved to the `javascript_cookies` table.
+    * Data is saved to the `cookies` table.
```
Comment (birdsarah) [inline README.md:152]: Is the cookies table a table of all cookies (including those set from http requests) or is it only those from JS.set_cookie?

Comment (englehardt) [reply README.md:152]: It's all cookies

Comment (birdsarah) [reply README.md:152]: I did not know that! Has it always been? (Just thinking about past data sources).

Comment (vringar) [reply README.md:152]: We just used to differentiate between `javascript_cookies` and `flash_cookies`, that's why there was any sort of prefix

**Pull request closed without merging**